js/main.js

```diff
@@ -23,38 +29,65 @@ function handleSubmit(event) {
     notes,
   };
 
-  formValues.entryId = data.nextEntryId;
-  data.nextEntryId++;
-  data.entries.unshift(formValues);
-
-  $photo.setAttribute('src', 'images/placeholder-image-square.jpg');
-
-  $form.reset();
-
   $ul.prepend(renderEntry('entries'));
   viewSwap($entryView);
 
   toggleNoEntries();
+
+  if (data.editing === null) {
+    renderEntry('entries');
```

Comment:
You need to append the result of `renderEntry` to your `ul`. Right now, you're creating an entry, but aren't actually adding it to the webpage